keyboards/lovelive9/keymaps/default/keymap.c

```diff
@@ -0,0 +1,370 @@
+  ),
+
+  [_RIKO] = LAYOUT(
+    KC_7, KC_8, KC_9, KC_PLUS, KC_MINS, KC_ASTR, RIKO, KC_SLSH, KC_LSFT
+  ),
+
+  [_MARI] = LAYOUT(
+    KC_TAB, KC_DEL, KC_COMM, KC_SPC, KC_ENT, KC_BSPC, KC_DOT, MARI, KC_LSFT
+  ),
+
+  [_KANAN] = LAYOUT(
+    KC_LEFT, KC_DOWN, KC_UP, KC_RGHT, KC_ENT, KC_BSPC, KC_AMPR, KC_KANJI, KANAN
+  ),
+  [_SCHOOL_IDOL_FESTIVAL] = LAYOUT(
+    KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9
+  ),
+};
+
+void check_tap_and_send_key(int MEMBER) {
+  if (long_tap_timer < 500) {
+    switch (MEMBER) {
+      case SEND_DIA:
+        SEND_STRING("KUROSAWA DIYA");
+      break;
+      case SEND_YOU:
+        SEND_STRING("WATANABE YOU");
+      break;
+      case SEND_TIKA:
+        SEND_STRING("TAKAMI TIKA");
+      break;
+      case SEND_ZURA:
+        SEND_STRING("KUNIKIDA HANAMARU");
+      break;
+      case SEND_RUBY:
+        SEND_STRING("KUROSAWA RUBY");
+      break;
+      case SEND_YOHANE:
+        SEND_STRING("TSUSHIMA YOSHIKO");
+      break;
+      case SEND_RIKO:
+        SEND_STRING("SAKURAUCHI RIKO");
+      break;
+      case SEND_MARI:
+        SEND_STRING("OHARA MARI");
+      break;
+      case SEND_KANAN:
+        SEND_STRING("MATSUURA KANAN");
+      break;
+      }
+  }
+  long_tap_timer = 0;
+}
+
+int aqours_color_h[] = { 26, 340, 150,   0, 199, 220, 53, 265, 322};
+int aqours_color_s[] = {255, 165, 255, 255, 255, 350, 255, 255, 255};
+int aqours_color_v[] = {255, 255, 255, 255, 255, 255, 200, 255, 255};
+
+void LED_default_set(void) {
+
+  sethsv(aqours_color_h[2], aqours_color_s[2], aqours_color_v[2], (LED_TYPE *)&led[0]);
+  sethsv(aqours_color_h[7], aqours_color_s[7], aqours_color_v[7], (LED_TYPE *)&led[1]);
+  sethsv(aqours_color_h[1], aqours_color_s[1], aqours_color_v[1], (LED_TYPE *)&led[2]);
+  sethsv(aqours_color_h[5], aqours_color_s[5], aqours_color_v[5], (LED_TYPE *)&led[3]);
+  sethsv(aqours_color_h[8], aqours_color_s[8], aqours_color_v[8], (LED_TYPE *)&led[4]);
+  sethsv(aqours_color_h[6], aqours_color_s[6], aqours_color_v[6], (LED_TYPE *)&led[5]);
+  sethsv(aqours_color_h[0], aqours_color_s[0], aqours_color_v[0], (LED_TYPE *)&led[6]);
+  sethsv(aqours_color_h[4], aqours_color_s[4], aqours_color_v[4], (LED_TYPE *)&led[7]);
+  sethsv(aqours_color_h[3], aqours_color_s[3], aqours_color_v[3], (LED_TYPE *)&led[8]);
+
+  rgblight_set();
+
+}
+
+
+void LED_layer_set(int aqours_index) {
+  for (int c = 0; c < 9; c++) {
+    sethsv(aqours_color_h[aqours_index], aqours_color_s[aqours_index], aqours_color_v[aqours_index], (LED_TYPE *)&led[c]);
+  }
+  rgblight_set();
+}
+
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+
+  switch (keycode) {
+
+    case DIA:
+      if (record->event.pressed) {
+        long_tap_timer = 1;
+        layer_on(_DIA);
+        LED_layer_set(3);
+      } else {
+        check_tap_and_send_key(SEND_DIA);
+        layer_off(_DIA);
+        LED_default_set();
+      }
+      return false;
+      break;
```

Comment:
All the `return false` statements in `process_record_user` supersede the `break` statements, so the `break` statements here are all redundant and can be deleted.

Reply:
thank you.
I will delete"return false" in switch~case